flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "require-important-update": true
```

Comment:
first time I see this...

Reply:
Yes, it's new and shiny, and very much fitting here.